include/vkgcDefs.h

```diff
@@ -71,6 +71,7 @@
 //* | -------- | ----------------------------------------------------------------------------------------------------- |
 //* |     40.0 | Added DescriptorReserved12, which moves DescriptorYCbCrSampler down to 13                             |
 //* |     39.0 | Non-LLPC-specific XGL code should #include vkcgDefs.h instead of llpc.h                               |
+//* |     38.3 | Added shadowDescriptorTableUsage and shadowDescriptorTablePtrHigh to PipelineOptions                  |
```

Comment:
That versioning is a bit odd, isn't it? :)

Reply:
I guess this is because we recently have two changes from Tim and Xue, the major version is updated and Carl doesn't catch up with this.

Reply:
I agreed with TimR to slot the version because XGL was still on 38 when I started this.  I can increase the version if appropriate, but the only impact would be this comment.